cmd/controllers/app/server.go

```diff
@@ -85,12 +85,12 @@ func Run(opt *options.ServerOption) error {
 	kbClient := kbver.NewForConfigOrDie(config)
 	vkClient := vkclient.NewForConfigOrDie(config)
 
-	jobController := job.NewJobController(kubeClient, kbClient, vkClient)
+	jobController := job.NewJobController(kubeClient, kbClient, vkClient, opt.WorkerThreads)
 	queueController := queue.NewQueueController(kubeClient, kbClient)
 	garbageCollector := garbagecollector.New(vkClient)
 
 	run := func(ctx context.Context) {
-		go jobController.Run(ctx.Done())
+		go jobController.Run(opt.WorkerThreads, ctx.Done())
```

Comment:
We already create jobController with `opt.WorkerThreads`, why `Run` still need this parameters?

Reply:
changed